### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by thomas-tu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - [case: 1203585] Removed `Custom Shape` option from the `Shape Editor` window.
 
+
+## [4.3.0-preview.3 - 2020-01-28
```
Comment (karl-) [inline CHANGELOG.md]: `]` you dropped this :)

Comment (thomas-tu) [reply CHANGELOG.md]: It doesn't deserve it

Comment (karl-) [inline CHANGELOG.md]:
```suggestion
## [4.3.0-preview.3] - 2020-01-28
```

Comment (thomas-tu) [reply CHANGELOG.md]: Fix in  3a6dfba

**Pull request merged**